src/Dev/BuildTask.php

```diff
@@ -82,4 +83,16 @@ public function getDescription()
     {
         return $this->description;
     }
+    
+    /**
+     * @return string  e.g. /dev/tasks/MyTask-In-Full
+     */
+    public function Link() : string
+    {
+        $link = $this->Config()->get('segment');
+        if (! $link) {
+            $link = str_replace('\\', '-', static::class);
+        }
+        return Director::absoluteUrl('dev/tasks/') . $link;
```

Comment:
I think the `dev/` and `tasks/` portions are, in theory, configurable via `SilverStripe\Dev\DevelopmentAdmin::registered_controllers` so I'm not sure we should be hardcoding it here. To be fair on the other site I haven't seen anyone ever change it.

There might be a way how to get the action (and its url) from the `SilverStripe\Dev\TaskRunner` controller, using/including the task name.